amqp-utils/src/main/java/io/enmasse/amqp/Artemis.java

```diff
@@ -146,7 +148,7 @@ private Message doAttribute(String resource, String attribute, Object ... parame
     }
 
     private Message doRequestResponse(Message message, Object ... parameters) throws TimeoutException {
-        return doRequestResponse(5, TimeUnit.SECONDS, message, parameters);
+        return doRequestResponse(10, TimeUnit.SECONDS, message, parameters);
```

Comment:
I can't help feeling that timeouts should come from config or env variables rather than only being changable by recompiling the code... however the issue is not new in this change :)

Reply:
Agreed, I will fix that because it might be quite useful.